backend/post-service/src/main/java/com/speakapp/postservice/services/PostService.java

```diff
@@ -56,6 +56,33 @@ public PostGetDTO createPost(PostCreateDTO postCreateDTO, UUID userId) {
         );
     }
 
+    public PostGetDTO updatePost(PostCreateDTO postCreateDTO, UUID postId, UUID userId) {
+        Post postUpdated = postRepository.getPostByPostId(postId);
```

Comment:
Gdy nie znajdziesz obiektu w bazie o takim postId to zwróci ci nulla. Konsekwencje tego bedą na dole.